src/ameba/rule/style/unless_else.cr

```diff
@@ -50,7 +50,30 @@ module Ameba::Rule::Style
     MSG = "Favour if over unless with else"
 
     def test(source, node : Crystal::Unless)
-      issue_for node, MSG unless node.else.nop?
+      return if node.else.nop?
+      return unless location = node.location
+      return unless cond_end_location = node.cond.end_location
+      return unless else_location = node.else_location
+      return unless end_location = node.end_location
```

Comment:
Hmm, lack of these shouldn't skip reporting the issue, they'd be just not correctable.